### ISSUE TITLE
Team sync - July 18, 2024

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#108]
Please take a moment to update the [Post PST A11y Audit](https://github.com/orgs/Quansight-Labs/projects/11/views/7) project board. 

Then please provide your updates using the following template. (You may have work outside the post-audit tasks, please include that work, too.)

- Progress made since last sync, if any
- Currently working on
- Blockers, questions, or anything the team can help you with
- Needs discussion
- Upcoming PTO
- How many hours per week you have allocated to this project

Previous: [Team sync - July 11, 2024](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/106)

**Comment by Carreau** [comment]
- Progress made since last sync, if any

A11y for light dark button, need review/thoughts from @isabela-pf for the text.

- Currently working on

^ Above and will work on collapsing UI.

- Blockers, questions, or anything the team can help you with
- Needs discussion

All good

- Upcoming PTO

None

- How many hours per week you have allocated to this project

Mixed 16h with 10h interactive docs and 6h maintenance and accessibility.

**Comment by isabela-pf** [comment]
- Progress made since last sync
     - Incorporated feedback into notebook authoring recommendations.
     - Posted rough outline for first event.
- Currently working on
     - Finalize dates and make logistic choices for events.
     - Add links elsewhere in authoring recommendations.
     - Remove glossary/checklist in authoring recommendations. 
- Blockers, questions, or anything the team can help you with
     - I don't have any right now, thanks.
- PTO
     - None upcoming.
- How many hours per week you have allocated to this project
     - 16

**Comment by gabalafou** [comment]
Minutes:

- Talked about the Labs all hands coming up on Tuesday
	- Question: is this about Scientific Python CZI grant or about anything accessibility at Quansight?
    	- I think it should be about anything accessibility
	- Gabriel to follow up on Slack
- Talked about [accessible names for the PST theme switcher](https://github.com/pydata/pydata-sphinx-theme/pull/1924) 
	- Isabela to follow up, but we don't want to spend too much time on this because it's not really critical for many screen reader users
	- Idea from Matthias (which I like a lot): perhaps make the theme switcher button open a dialog that lets the user choose which color scheme they want, but also informs them how to change the color scheme at the browser and/or system level.
- Discussion of hover effect on collapsible admonitions
	- Sought clarification from Isabela if this referred to just the Sphinx-togglebutton admonition or all of the collapsible admonitions. Response: none of them had a clear enough hover effect.
	- Decisions made: 
		- Consistency between all admonitions is desired
		- Underline is enough of a hover effect. (Could even underline the chevron icon, if necessary.)
- Around the room updates
